Tools/dea_tools/dask.py

```diff
@@ -41,44 +46,46 @@ def create_local_dask_cluster(spare_mem="3Gb", display_client=True, return_clien
         sys.path.append("../Scripts")
         from dea_dask import create_local_dask_cluster
 
-        create_local_dask_cluster(spare_mem='4Gb')
+        create_local_dask_cluster()
 
     Parameters
     ----------
-    spare_mem : String, optional
-        The amount of memory, in Gb, to leave for the notebook to run.
-        This memory will not be used by the cluster. e.g '3Gb'
     display_client : Bool, optional
         An optional boolean indicating whether to display a summary of
         the dask client, including a link to monitor progress of the
         analysis. Set to False to hide this display.
     return_client : Bool, optional
         An optional boolean indicating whether to return the dask client
         object.
+    n_workers : int, optional
+        Number of workers to start, default is set to 1 which works well
+        with loading ODC data.
+    threads_per_worker: int, optional
+        Number of threads per each worker, by default this will be set to
+        the number of cpus on the machine.
 
     """
-    # Attempt to import datacube and raise an error if not available
-    try:
-        from datacube.utils.dask import start_local_dask
-        from datacube.utils.rio import configure_s3_access
-    except ImportError as e:
-        raise ImportError(
-            "`datacube` is required for `create_local_dask_cluster`. "
-            "Please install DEA Tools with the `[datacube]` extra, e.g.: "
-            "`pip install dea-tools[datacube]`"
-        ) from e
 
     if _HAVE_PROXY:
         # Configure dashboard link to go over proxy
         prefix = os.environ.get("JUPYTERHUB_SERVICE_PREFIX", "/")
         dask.config.set({"distributed.dashboard.link": prefix + "proxy/{port}/status"})
 
-    # Start up a local cluster
-    client = start_local_dask(mem_safety_margin=spare_mem)
-
+    # count cpus if threads_per_worker not provided
+    if threads_per_worker is None:
+        if get_cpu_quota() is not None:
+            threads_per_worker = round(get_cpu_quota())
+        else:
+            threads_per_worker = os.cpu_count()
+
+    # start client
+    client = dask.distributed.Client(
+        n_workers=n_workers, threads_per_worker=threads_per_worker
+    )
+    
+    # configure aws access
     if _IS_AWS:
```

Comment:
I wonder if this is broadly applicable outside of the Sandbox (e.g. if a user is running code locally or in Github Codespaces etc - I might ask whether there's any reason for this check

Reply:
I'm not really sure about this. Only included because it was there in the old function so I assumed it must be checking some environment variable.